spec/services/active_record_utils_spec.rb

```diff
@@ -2,9 +2,6 @@
 
 require 'rails_helper'
 
-require 'active_record_utils'
-require 'audit_results'
-
```

Comment:
This is dead code, vestiges of when this code used to live in `lib/`. Why? Because code in `lib/` is not auto-loaded by Rails apps, and all code in `app/` is.